.circleci/config.yml

```diff
@@ -423,7 +423,7 @@ jobs:
     resource_class: medium
     parallelism: 4
     environment:
-      SKIP_E2E_SUBS: "true"
+      E2E_TEST_FILTER: "GO"
```

Comment:
weird, why is this change shown?

Reply:
I re-merged it from the origin/master instead of from the master fork.. which seems to resolve the issue.